components/server/ee/src/prebuilds/bitbucket-app.ts

```diff
@@ -156,12 +156,14 @@ export class BitbucketApp {
                 { span },
                 { user, project: projectAndOwner.project, context, commitInfo },
             );
-            await this.webhookEvents.updateEvent(event.id, {
-                prebuildStatus: "prebuild_triggered",
-                status: "processed",
-                prebuildId: ws.prebuildId,
-            });
-            return ws;
+            if (!ws.done) {
+                await this.webhookEvents.updateEvent(event.id, {
+                    prebuildStatus: "prebuild_triggered",
+                    status: "processed",
+                    prebuildId: ws.prebuildId,
+                });
+                return ws;
+            }
```

Comment:
I'm not sure I fully understand this change. What does it mean when `ws.done` is `true`? And does it always mean that we did not in fact trigger a prebuild? 💭